slice/Ice/OperationMode.ice

```diff
@@ -27,18 +27,22 @@ module Ice
         /// Ordinary operations have <code>Normal</code> mode. These operations modify object state; invoking such an
         /// operation twice in a row has different semantics than invoking it once. The Ice run time guarantees that it
         /// will not violate at-most-once semantics for <code>Normal</code> operations.
+        ["swift:identifier:normal"]
         Normal,
 
         /// Operations that are <code>nonmutating</code> must not modify object state.
         /// The Ice run-time no longer makes a distinction between nonmutating operations and idempotent operations.
         /// Use the <code>idempotent</code> keyword instead.
-        ["deprecated:Use Idempotent instead."] Nonmutating,
+        ["deprecated:Use Idempotent instead."]
+        ["swift:identifier:nonMutating"]
```

Comment:
Nonmutating is a single word, so the mapped identifier should be nonmutating, like the old Slice keyword.